docs/design/fabric-v2+/fpc-management.md

```diff
@@ -0,0 +1,80 @@
+# FPC Management
+
+This document defines the management APIs of FPC.
+Such definition builds on the design diagrams describing the enclave creation, 
+the enclave registration and the chaincode key generation.
+
+[//]: # (## Enclave and Chaincode APIs)
+
+[//]: # (### Create Chaincode Enclave)
+
+[//]: # (### Register Chaincode Enclave)
+
+[//]: # (### Generate FPC Chaincode Keys)
```

Comment:
What is this? Can not see anything rendered from this.

Reply:
removed -- those were comments